org-formation/organization.yaml

```diff
@@ -313,20 +312,6 @@ Organization:
       Tags:
         <<: !Include ./_default_org_tags.yaml
 
-  # INCLUDE is the grant where we collaborate with CHOP
-  IncludeChopProdAccount:
```

Comment:
Removing accounts from this file will orphan accounts from org-formation which means that org-formation can no longer deploy/remove stuff to these aws accounts. You need to create a separate PR for this and merge it at the very end of your PR chain.